cmd/get/execution.go

```diff
@@ -0,0 +1,58 @@
+package get
+
+import (
+	"context"
+	"github.com/golang/protobuf/proto"
+	"github.com/lyft/flytectl/cmd/config"
+	cmdCore "github.com/lyft/flytectl/cmd/core"
+	"github.com/lyft/flytectl/pkg/printer"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+)
+
+var executionColumns = []printer.Column{
+	{"Version", "$.id.version"},
+	{"Name", "$.id.name"},
+	{"LaunchPlan", "$.spec.launchplan.name"},
+	{"Phase", "$.spec.phase"},
+	{"Duration", "$.closure.duration"},
+	{"StartedAt", "$.closure.started_at"},
+	{"Workflow", "$.closure.workflow_id.name"},
+	{"Metadata", "$.spec.metadata"},
+}
+
+var executionSingleColumns = []printer.Column{
+	{"TaskID", "$.id.taskID"},
+	{"NodeExecutionID", "$.id.nodeExecutionID"},
+	{"Phase", "$.spec.phase"},
+	{"Duration", "$.closure.duration"},
+	{"StartedAt", "$.closure.started_at"},
+}
+
+func ExecutionToProtoMessages(l []*admin.Execution) []proto.Message {
+	messages := make([]proto.Message, 0, len(l))
+	for _, m := range l {
+		messages = append(messages, m)
+	}
+	return messages
+}
+
+func getExecutionFunc(ctx context.Context, args []string, cmdCtx cmdCore.CommandContext) error {
```

Comment:
nit: looks like you're calling the List endpoint in the method body, can you update the function name (get -> list)?